### PR TITLE
New function - decapitalizeFirst

### PR DESCRIPTION
**Pull request by lukasduspiva**

### CODE REVIEW

File: packages/ramda-extension/src/decapitalizeFirst.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { replace, toLower, unless, isNil } from 'ramda';
+
+/**
+ * For not-nil string returns string which starts with lower-case letter.
+ *
+ * @func
+ * @category String
+ *
+ * @param {any} x Any string
```
Comment (tommmyy) [inline packages/ramda-extension/src/decapitalizeFirst.js]: `String`

File: packages/ramda-extension/src/decapitalizeFirst.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * @return {any} Description
```
Comment (tommmyy) [inline packages/ramda-extension/src/decapitalizeFirst.js]: missing description

**Comment by lukasduspiva** [comment]
Oops, I will fix these later.

File: packages/ramda-extension/src/decapitalizeFirst.js (hunk 2 above)
Comment (lukasduspiva) [reply packages/ramda-extension/src/decapitalizeFirst.js]: Fixed.

File: packages/ramda-extension/src/decapitalizeFirst.js (hunk 1 above)
Comment (lukasduspiva) [reply packages/ramda-extension/src/decapitalizeFirst.js]: Fixed.

**Pull request merged**